src/ui/Pagination.tsx

```diff
@@ -98,12 +102,22 @@ const Pagination = ({
           z-index: ${getValue('zIndex')};
         }
 
-        .prev-btn {
-          margin-right: 0.8rem;
+        .prev-btn.disabled > span {
+        background-color: ${udbMainLightGrey};
```

Comment:
Normally you shouldn't import colors directly.

If I'm not mistaken you can get to the color by doing

`background-color: ${({ theme }) => theme.colors.udbMainLightGrey};`

Reply:
```ts
const getColor = getValueFromTheme('colors');
`background-color: ${getColor('udbMainLightGrey')}`
```